src/main/java/org/junit/runners/model/FrameworkMethod.java

```diff
@@ -149,6 +149,11 @@ public boolean isShadowedBy(FrameworkMethod other) {
     }
 
     @Override
+    boolean isBridgeMethod() {
```

Comment:
@sormuras synetetic methods are any methods introduced by the compiler that do not correspond to something in the source (with the exception of default constructors and static initializers). Therefore any bridge method is a synthetic method.